src/annotation_processor.md

```diff
@@ -0,0 +1,149 @@
+---
+layout: guide
+title: PDL Schema
+permalink: /annotation_processor
+excerpt: Documentation of using annotation processor in Rest.Li
+---
+
+# Annotation Processor
+This sections introduces a generic schema annotation processing tool in Rest.Li. Before reading this, it is recommended to be familiar with the concept of PathSpec in Rest.Li. [Insert link here]
+
+## Annotating Rest.Li Schemas
+In Rest.Li, schema writers are able to add annotations to schema fields or the schema itself.
```

Comment:
The way you say "schema writers" sounds impersonal and almost implies that the "writers" are Java Writer classes. I would say "authors"